spotbugs/src/main/java/edu/umd/cs/findbugs/OpcodeStack.java

```diff
@@ -2633,6 +2636,36 @@ private void processMethodCall(DismantleBytecode dbc, int seen) {
                 return;
             }
             push(requestParameter); // fall back to standard logic
+        } else if (seen == Const.INVOKESTATIC &&
```

Comment:
These three could be refactored into one, since the only differences are the string values. Perhaps turn the three new constants into a map from `clsName` to signature, check if `clsName` is one of the keys, and if so, look up the value.